RosBE-Windows/Root/ChangeLog.txt

```diff
@@ -1,5 +1,23 @@
-*** Feb 01th, 2018 - RosBE 2.1.6 Released
-
+*** Aug 17th, 2019 - RosBE 2.2.0 Released
+
+- Updated GCC to 8.3.0 (Colin Finck, Daniel Reimer)
+  - i686-w64-mingw32 AND x86_64-w64_mingw32 (optional) in one setup!
+- Updated binutils to 2.32 (Colin Finck, Daniel Reimer)
+- Updated mingw-w64 to 6.0.0 (Colin Finck, Daniel Reimer)
+- Updated Flex to 2.6.4 (Daniel Reimer) (https://sourceforge.net/projects/winflexbison/)
```

Comment:
It's Bison 3.5 and Flex 2.6.4+ (revision 8b1fbf674f2e038df9cf1fe7725617e3837ae2a9) with my updated binaries from https://svn.reactos.org/RosBE-Temp/RosBE-2.2_Dec2019.7z
We are using the very same versions for RosBE-Unix now and can therefore safely make Bison and Flex a build dependency.

The winflexbison sources are heavily modified to an extent, where they don't build on anything else than Windows/MSVC anymore and many code paths needed to be disabled for this.
Therefore, winflexbison wasn't suitable for sourcing reliable versions that can be shared between RosBE-Windows and RosBE-Unix

Reply:
Fixed I hope.